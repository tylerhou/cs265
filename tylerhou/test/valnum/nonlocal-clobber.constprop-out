@main {
.__b0:
  x: int = const 1;
.lb:
  y: int = const 1;
  x: int = const 2;
  print y;
}
