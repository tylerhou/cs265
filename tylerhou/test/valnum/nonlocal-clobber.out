@main {
.__b0:
  x: int = const 1;
.lb:
  y: int = id x;
  x: int = add x x;
  print y;
}
